node_modules/oae-authentication/lib/api.js

```diff
@@ -16,23 +16,25 @@
 var _ = require('underscore');
 var events = require('events');
 var passport = require('passport');
-
+var crypto = require('crypto');
```

Comment:
`npm install --save crypto` adds the package to `package.json` directly beside installing it.